code/game/machinery/doors/shutters.dm

```diff
@@ -10,6 +10,9 @@
 	max_integrity = 100
 	recipe_type = /datum/crafting_recipe/shutters
 
+/obj/machinery/door/poddoor/shutters/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+	return NONE
```

Comment:
It looks like you made the other changes but kept this in?

Reply:
I missed that when I was moving the other procs around.  Removed with cde9c19b280cf6049bb7b23c356a57172c147494